torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+from torch.fx.graph_module import GraphModule
+from typing import Any
+from torch.fx.node import Node
+
+def get_all_users_of(fx_module: GraphModule, index: int) -> List[int]:
+    """Given the graph(fx_module) and an index, return a list of all node indexes that use this node"""
+    """find_use is used to find out if the node is another node's arg or kwargs."""
+    def find_use(arg: Any) -> bool:
+        if isinstance(arg, (tuple, list)):
+            return any(find_use(elem) for elem in arg)
+        elif isinstance(arg, dict):
+            return any(find_use(v) for k, v in arg.items())
+        elif isinstance(arg, slice):
+            return any(find_use(arg.start), find_use(arg.stop), find_use(arg.step))
+        elif isinstance(arg, Node):
+            return arg is current_node
+        else:
+            return False
+    graph = fx_module.graph
+    current_node = graph.nodes[index]
+    user_indexes: List[int] = []
+    """if the node A is in node B's args, then B is the user of A
+       go through all the nodes, if the input node in any node's args,
+       then that node is the input node's user
+    """
+    for i, n in enumerate(graph.nodes):
+        if find_use(n.args) or find_use(n.kwargs):
+            user_indexes.append(i)
+    return user_indexes
```

Comment:
Oh one more issue is that this won't inform the user of uses of a `Node` in the output of the GraphModule. I'm working on https://github.com/pytorch/pytorch/pull/45245 to make the handling of outputs more uniform